modules/ROOT/pages/clustering/setup/deploy.adoc

```diff
@@ -130,15 +148,106 @@ SHOW SERVERS YIELD *;
 | "73e9a990-0a97-4a09-91e9-622bf0b239a4" | "73e9a990-0a97-4a09-91e9-622bf0b239a4" | "server03:7687" | "Enabled" | "Available" | ["system", "neo4j"] | ["system", "neo4j"] | []   | []               | []              | "NONE"         | "5.0.0"     |
 +-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 ----
+
+[TIP]
+.Startup time
+====
+The instance may appear unavailable while it is joining the cluster.
+If you want to follow along with the startup, you can see the messages in xref:configuration/file-locations.adoc[_neo4j.log_].
 ====
 
+======
+[role=include-with-Discovery-service-v2 label--new-5.22]
+======
+
+
+In this example, three servers named `server01.example.com`, `server02.example.com` and `server03.example.com` are configured.
+Neo4j Enterprise Edition is installed on all three servers.
+They are configured by preparing xref:configuration/file-locations.adoc[_neo4j.conf_] on each server.
+Note that they are all identical, except for the configuration of `server.default_advertised_address`.
+To use discovery service v2, you have to set the configuration of `dbms.cluster.discovery.version` to `V2_ONLY`.
+Instead of `dbms.cluster.discovery.endpoints`, use `dbms.cluster.discovery.v2.endpoints`.
+
+
+._neo4j.conf_ on server01.example.com:
+[source, properties]
+----
+server.default_listen_address=0.0.0.0
+server.default_advertised_address=server01.example.com
+dbms.cluster.discovery.v2.endpoints=server01.example.com:5000,server02.example.com:5000,server03.example.com:5000
```

Comment:
The default port for V2 discovery is `6000` instead of `5000`, so I think each of these need changing.
```suggestion
dbms.cluster.discovery.v2.endpoints=server01.example.com:6000,server02.example.com:6000,server03.example.com:6000
```